docs/source/en/model_doc/llava.md

```diff
@@ -43,13 +43,13 @@ The original code can be found [here](https://github.com/haotian-liu/LLaVA/tree/
 - For better results, we recommend users to prompt the model with the correct prompt format: 
 
 ```bash
-"USER: <image>\n<prompt>ASSISTANT:"
+"USER: <image>\n<prompt> ASSISTANT:"
```

Comment:
Technically we should prepend the system prompt here, but I think it can be excluded for the purposes of keeping the demo simple.